synapseclient/models/services/storable_entity_components.py

```diff
@@ -120,6 +120,77 @@ async def store_entity_components(
     return re_read_required
 
 
+async def store_entity_components_file_folder_only(
```

Comment:
This was a hack to make sure that storing files/folders attached to the `project` and `folder` class could still be saved. See the logic for `annotations` and `activity` is commented out in this code.